src/Metadata.aspx.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+
+namespace GroupDocs.Metadata.WebForms
+{
+    public partial class Metadata : System.Web.UI.Page
+    {
+        protected void Page_Load(object sender, EventArgs e)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add a nested comment explaining why this method is empty, throw a 'NotSupportedException' or complete the implementation.](https://app.codacy.com/gh/groupdocs-metadata/GroupDocs.Metadata-for-.NET-WebForms/pullRequest?prid=5102156)